### PR TITLE
Region not required when removing it

### PR DESCRIPTION
**Pull request by h0lyalg0rithm**

### CODE REVIEW

File: plugin/wavesurfer.regions.js [hunk 1]
```diff
@@ -461,6 +461,10 @@ WaveSurfer.clearRegions = function () {
     this.regions && this.regions.clear();
 };
 
+WaveSurfer.clearRegion = function (region) {
+    this.regions && region.remove();
+};
```
Comment (katspaugh) [inline plugin/wavesurfer.regions.js]: I don't think such a method is useful. One can simply call `region.remove()`.

File: plugin/wavesurfer.regions.js [hunk 2]
```diff
@@ -169,7 +169,7 @@ WaveSurfer.Region = {
     },
 
     /* Remove a single region. */
-    remove: function (region) {
+    remove: function () {
```
Comment (katspaugh) [inline plugin/wavesurfer.regions.js]: 👍

**Comment by h0lyalg0rithm** [comment]
I will push a commit without the helper method.

**Comment by katspaugh** [comment]
Thank you, merged.

**Pull request merged**